tasks/eva_2156/deprecate_variant_from_contig.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+from copy import deepcopy
+from datetime import datetime
+
+import pymongo
+
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+
+
+def inactive_object(variant):
+    inactive_variant = deepcopy(variant)
+    inactive_variant['hashedMessage'] = inactive_variant.pop('_id')
+    return {
+        "eventType": "DEPRECATED",
+        "accession": variant['accession'],
```

Comment:
My hope is that we can count on PyMongo to preserve the fidelity of this numeric field to NumberLongs (both here and in the inner inactiveObjects document) as in
![image](https://user-images.githubusercontent.com/25152678/95733573-3df5e280-0c7a-11eb-9216-37c14e807a23.png).... You can verify this after running the script...

Reply:
I had a quick check and can confirm that it is the case.